src/client/datascience/interactive-ipynb/nativeEditorCommandListener.ts

```diff
@@ -99,19 +99,21 @@ export class NativeEditorCommandListener implements IDataScienceCommandListener
         }
     }
 
-    private interruptKernel(document: Uri | undefined) {
+    private interruptKernel(notebookUri: Uri | undefined) {
```

Comment:
Renamed poorly named args, very confusing.
In the tests we were passing `document` and here we're expecting uri.